packages/client/src/clients/in-app-messages/message-client.ts

```diff
@@ -69,75 +73,89 @@ export class InAppMessagesClient {
       return;
     }
 
-    // TODO: Move to method on channel client
     // Set the loading type based on the request type it is
-    this.channelClient.store.setState((state) => ({
-      ...state,
-      queries: {
-        ...state.queries,
-        [this.queryKey]: {
-          ...queryState,
-          networkStatus: options.__loadingType ?? NetworkStatus.loading,
-          loading: true,
-        },
-      },
-    }));
-
-    // TODO: Move to method on user.getInAppMessages
-    const result = await this.knock.client().makeRequest({
-      method: "GET",
-      url: `/v1/users/${this.knock.userId}/in-app-messages/${this.channelClient.channelId}/${this.messageType}`,
-      params: queryParams,
+    this.channelClient.setQueryState(this.queryKey, {
+      ...queryState,
```

Comment:
I feel like this helper should be made into one that updates a query status only. Which means, 
* `...queryState` part should be done as part of the helper
* the helper to be renamed to something like `setQueryStatus`